### PR TITLE
Test mobile push

### PR DESCRIPTION
**Pull request by Konstantinov-Innokentii**
# What this PR does
Adds ability to send test push notification

### CODE REVIEW

File: engine/apps/mobile_app/demo_push.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    message = _get_test_escalation_fcm_message(user, device_to_notify, critical)
+    _send_push_notification(device_to_notify, message)
+
+
+def _get_test_escalation_fcm_message(user: User, device_to_notify: FCMDevice, critical: bool) -> Message:
+    # TODO: this method is copied from _get_alert_group_escalation_fcm_message
+    # to have same notification/sound/overrideDND logic. Ideally this logic should be abstracted, not repeated.
+    from apps.mobile_app.models import MobileAppUserSettings
+
+    thread_id = f"test_push"
```
Comment (vadimkerr) [inline engine/apps/mobile_app/demo_push.py:21]: Maybe add some randomness (e.g. random uuid) to make sure notifications are not grouped?

File: engine/apps/mobile_app/demo_push.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+from apps.user_management.models import User
+
+
+def send_test_push(user, critical=False):
+    device_to_notify = FCMDevice.objects.filter(user=user).first()
+    message = _get_test_escalation_fcm_message(user, device_to_notify, critical)
+    _send_push_notification(device_to_notify, message)
+
+
+def _get_test_escalation_fcm_message(user: User, device_to_notify: FCMDevice, critical: bool) -> Message:
```
Comment (vadimkerr) [inline engine/apps/mobile_app/demo_push.py:16]: Consider adding a unit test that checks that FCM message values, something like [this](https://github.com/grafana/oncall/blob/49d2f477c7064c06651ff6cbe54760515f5c5cdc/engine/apps/mobile_app/tests/test_notify_user.py#L215-L243)

File: engine/apps/api/views/user.py [hunk 3]
```diff
@@ -391,6 +394,20 @@ def make_test_call(self, request, pk):
 
         return Response(status=status.HTTP_200_OK)
 
+    @action(detail=True, methods=["post"], throttle_classes=[TestCallThrottler])
+    def send_test_push(self, request, pk):
```
Comment (vadimkerr) [inline engine/apps/api/views/user.py:398]: Consider adding a unit test for this endpoint, something like [this](https://github.com/grafana/oncall/blob/62412076a2f83ccf7ab028938d81d17b6f5b9d02/engine/apps/mobile_app/tests/test_user_settings.py#L7-L34)

File: engine/apps/api/views/features.py [hunk 4]
```diff
@@ -14,6 +14,7 @@
 FEATURE_GRAFANA_CLOUD_CONNECTION = "grafana_cloud_connection"
 FEATURE_WEB_SCHEDULES = "web_schedules"
 FEATURE_WEBHOOKS2 = "webhooks2"
+FEATURE_MOBILE_TEST_PUSH = "mobile_test_push"
```
Comment (joeyorlando) [inline engine/apps/api/views/features.py:17]: what is this for? is this being used anywhere?

**Pull request merged**